src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,31 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public class CEO {
+    public CEO(String name)
+    {
+        this.name=name;
+    }
+    private String name;
```

Comment:
If a field cannot be modified (and the CEO's name should not be modified) it can be declared as `final` :)